src/displayapp/screens/Weather.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include <cstdint>
+#include <lvgl/lvgl.h>
+#include "displayapp/screens/Screen.h"
+#include "components/ble/SimpleWeatherService.h"
+#include "displayapp/apps/Apps.h"
+#include "displayapp/Controllers.h"
+#include "Symbols.h"
+#include "utility/DirtyValue.h"
+
+namespace Pinetime {
+
+  namespace Controllers {
+    class Settings;
+  }
+
+  namespace Applications {
+    namespace Screens {
+
+      class Weather : public Screen {
+      public:
+        std::array<const char*, 7> WeekDays = {"Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"};
```

Comment:
1. There is something like this in the DatetimeController. Maybe you can use this instead so we don't have multiple definitions of the days names.
2. Why is this public?

Reply:
Fixed.